### PR TITLE
Project documentation fix

### PR DESCRIPTION
**Pull request by septimium**
### Pull Request Overview

I changed the color of the arrows from white to black in 'Architecture.png' because they were invisible in light mode. Commit message is wrong, I wanted to write light mode instead of dark mode.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Mihai-Septimiu Tigaeru <septimiutg@gmail.com>

### CODE REVIEW

File: docs/project/2024/mihai.tigaeru/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+| [LED Display Module with Serial Interface (TM1637)](https://robojax.com/learn/arduino/robojax-TM1637_display_manual.pdf) | Score Counting | [11 RON](https://www.sigmanortec.ro/modul-display-led-cu-interfata-seriala-tm1637) |
+| [Dupont Wires M-M](https://www.farnell.com/datasheets/3178883.pdf) | Connections | [7 RON](https://www.bitmi.ro/componente-electronice/40-x-fire-dupont-tata-tata-20cm-10511.html) |
+| [Dupont Wires F-F](https://www.farnell.com/datasheets/3178883.pdf) | Connections | [6 RON](https://www.bitmi.ro/componente-electronice/40-x-fire-dupont-mama-mama-20cm-10509.html) |
+| [Dupont Wires M-F](https://www.farnell.com/datasheets/3178883.pdf) | Connections | [6 RON](https://www.bitmi.ro/componente-electronice/40-x-fire-dupont-tata-mama-20cm-10512.html) |
+| [Jumper Wires Kit](https://www.farnell.com/datasheets/3178883.pdf) | Connections | [9 RON](https://ardushop.ro/ro/home/1374-set-jumper-breadboard-140.html) |
+| [Micro USB Cable](https://www.techtarget.com/whatis/definition/micro-USB) | Power | [8 RON](https://ardushop.ro/ro/electronica/397-cablu-usb-micro.html?search_query=micro+usb&results=383) |
+
+## Software
+
+| Library | Description | Usage |
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.tigaeru/index.md:68]: No libraries?

**Review by alexandruradovici — DISMISSED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Please resolve the conflicts, a rebase would and `git push --force` would help.

**Comment by septimium** [comment]
Where exactly should I use those commands? In which repo?

**Comment by alexandruradovici** [comment]
> Where exactly should I use those commands? In which repo?

https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request

**Comment by septimium** [comment]
I will try something else

**Comment by septimium** [comment]
I think it should work now.

File: ername
Comment (alexandruradovici) [inline ername:1]: Please delete this file.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**